tests/vmcheck/overlay.sh

```diff
@@ -84,6 +84,7 @@ rsync -rlv $INSTTREE/usr/ vmcheck/usr/
 if [ -d $INSTTREE/etc ]; then # on CentOS, the dbus service file is in /usr
   rsync -rlv $INSTTREE/etc/ vmcheck/usr/etc/
 fi
+chcon -t install_exec_t vmcheck/usr/{bin/{rpm-,}ostree,libexec/rpm-ostreed}
```

Comment:
I think what we really need here is `ostree commit --selinux-label`.

Reply:
Agreed, that makes sense. I opened https://github.com/ostreedev/ostree/issues/1113.